Assets/Scripts/Managers/BuddyManager.cs

```diff
@@ -6,6 +6,9 @@ public class BuddyManager : SingletonBehaviour<BuddyManager> {
 
 	List<BuddyStats> _buddyStats = new List<BuddyStats>();
 
+	[SerializeField] CheckpointLifter[] checkpointLifters = new CheckpointLifter[1]; // Need these to activate checkpoints
+	int checkpointIndex = 0; // Index of next checkpoint to activate
```

Comment:
consider making this ReadOnly so we can see what lifter we're on while debugging